src/data-loader.ts

```
@@ -3556,4 +3655,194 @@
+			});
+
+			vi.stubEnv('CLAUDE_CONFIG_DIR', `${fixture.path},${fixture.path}`);
+
+			const paths = getClaudePaths();
+			const normalizedFixture = path.resolve(fixture.path);
+			// Should only contain the fixture path once (but may include defaults)
+			const fixtureCount = paths.filter(p => p === normalizedFixture).length;
+			expect(fixtureCount).toBe(1);
+		});
+
+		it('handles directory existence checks deterministically with mocked isDirectorySync', async () => {
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)